docs/content/preview/integrations/mirantis.md

```diff
@@ -0,0 +1,223 @@
+
+<i class="icon-debian"></i> [Docker for Debian](https://store.docker.com/editions/community/docker-ce-server-debian)
+
+## Install and configure Mirantis
+
+1. Install MKE docker image as follows:
+
+    ```sh
+    docker image pull mirantis/ucp:3.5.8
+    ```
+
+    ```sh
+    docker container run --rm -it --name ucp \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          mirantis/ucp:3.5.8 install \
+          --host-address <node-ip> \ // Replace <node-ip> with the IP address of your machine.
+          --interactive
+    ```
+
+    When prompted, enter the `username` and `password` that you want to set which will be used to access MKE web UI.
+
+1. Install and configure kubectl with MKE, and install Helm using the instructions in [MKE documentation](https://docs.mirantis.com/mke/3.5/ops/access-cluster/configure-kubectl.html?highlight=kubectl).
+
+1. Create a new storage class using the following steps:
+
+    1. ​​Copy the following content to a file named `storage.yaml`:
+
+        ```conf
+        apiVersion: storage.k8s.io/v1
+        kind: StorageClass
+        metadata:
+         name: yb-storage
+        provisioner: kubernetes.io/no-provisioner
+        volumeBindingMode: WaitForFirstConsumer
+        ```
+
+    1. Apply the configuration using the following command:
+
+        ```sh
+        kubectl apply -f storage.yaml
+        ```
+
+1. Make the new storage class default as follows:
+
+    ```sh
+    kubectl patch storageclass yb-storage -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+    ```
+
+1. Verify that the storage class is created using the following command:
+
+    ```sh
+    kubectl get storageclass
+    ```
+
+    ```output
+    NAME                 PROVISIONER                   RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+    yb-storage(default)  kubernetes.io/no-provisioner  Delete          WaitForFirstConsumer   false                  23s
+    ```
+
+1. Create four [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) using the following steps:
+
+    1. Copy the following PersistentVolume configuration to a file `volume.yaml`:
+
+        ```conf
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          name: task-pv-volume1
+          labels:
+            type: local
+        spec:
+          storageClassName: yb-storage
+          capacity:
+            storage: 10Gi
+          accessModes:
+             - ReadWriteOnce
+          hostPath:
+            path: "/mnt/data"
+        ```
+
+    1. Apply the configuration using the following command:
+
+        ```sh
+        kubectl apply -f volume.yaml
+        ```
+
+    1. Repeat the preceding two steps to create the remaining PersistentVolumes (PV) : `task-pv-volume2`, `task-pv-volume3`, and `task-pv-volume4` by changing the metadata name in `volume.yaml` for each volume.
```

Comment:
acronym defined but not the first use and not used elsewhere

all these added PVs are added to the same volume.yaml file?

Reply:
I thought so . I actually had the same doubt. @HarshDaryani896 - can you clarify this please ?

Reply:
so, change the metadata name and re-run the command on the same file, x3